main.tf

```diff
@@ -34,6 +36,14 @@ output "web_acl_id" {
 
 locals {
     name_underbar = replace(var.name, "-", "_")
+
+    ip_sets_rule = var.ip_set != null ? [ {
+      name       = "${var.ip_set_action}_ips"
```

Comment:
I can't be certain, but I feel that having a "IP set rule" with the name of "block_ips" or "allow_ips" might collide when the same rules get used for dev, qa, staging and prod.

If they're tied to a particular WAF "namespace" then it should be fine as the WAF has the stage in it.  Thoughts?

Reply:
test

Reply:
I've updated to have a more specific name